README.md

```diff
@@ -1,72 +1,185 @@
-To facilitate pull requests, a branch will be created for each of the major versions accepting support. For example,
-should we have tags denoting the versions, `v11.1.0`, `v11.2.0`, and `v12.0.0`, a branch `v11` would be present for submission
-of PRs.
-
-## Azure Versions
-Azure services _mostly_ do not use SemVer based versions. Rather, they stamp a set of REST endpoints with a date identifier. One will often
-see these stamps casually referred to as "API Versions". At the moment, our SDK only supports the most recent stamp for each service. In
-order to lock to an API version, one must also lock to an SDK version. However, as discussed in [#517](https://github.com/Azure/azure-sdk-for-go/issues/517),
-our objective is to reorganize and publish an independent package for each stamped API version of each service. In that way, we'll be able to support all API Versions
-in a single SDK Version.
-
-Knowing which API Versions of services are compatbile with one another, and finding which API Versions are available in which environments
-has been a common source of frustration for users. Along with Azure Stack, these problems have led to the development of "Profiles" which are
-aggregations of multiple services at particular API Versions. Using profiles with our SDK will be optional, and to opt-in you will need to be
-running Go 1.9 or higher.
-
-# Documentation
-
-- Azure SDK for Go Documentation is available at [GoDoc.org](http://godoc.org/github.com/Azure/azure-sdk-for-go/).
-- Azure REST APIs used by packages in this repository are documented at [Microsoft Docs, Azure REST](https://docs.microsoft.com/en-us/rest/api/).
-- Azure Services are discussed in detail at [Microsoft Docs, Azure Services](https://docs.microsoft.com/en-us/azure/#pivot=services).
-
-# Code samples
-
-- [Getting Started with Azure Blob Service in Go](https://github.com/Azure-Samples/storage-blob-go-getting-started)
+
+*More info in [the wiki](https://github.com/Azure/azure-sdk-for-go/wiki/Versioning)*
+
+You'll need to consider both SDK and API versions when selecting your
+targets. The following will describe them one by one.
+
+**SDK versions** are set with repository-wide [tags][]. We try to adhere to
+[semver][], so although we've long moved past version zero, we continue to
+add `-beta` to versions because **this package is still in preview.**
+
+[tags]: https://github.com/Azure/azure-sdk-for-go/tags
+[semver]: https://semver.org
+
+Azure **API versions** are typically a date string of form
+`yyyy-mm-dd[-preview|-beta]`. Whatever SDK version you use, you must also
+specify an API version in your import path, for example:
+
+```go
+import "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2016-03-01/compute"
+import "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2017-09-01/network"
+```
+
+See [SERVICES.md][] for a full list of services and paths.
+
+Azure **API profiles** specify a set of API versions tested for
```

Comment:
Maybe specify that you can use `latest` for the latest stable api versions and `preview` for the latest preview api versions